### ISSUE TITLE
build error v3.7.3 with wolfssl

### ISSUE THREAD

**Issue by linuxmaniac** [SIPp/sipp#750]
while trying to package the new version I'm having this error building for Debian sid

```
/build/sip-tester-3.7.3=. -fstack-protector-strong -fstack-clash-protection -Wformat -Werror=format-security -fcf-protection -Wdate-time -D_FORTIFY_SOURCE=2 -std=c++14 -Wall -Werror -pedantic -Wno-deprecated-co│
py -Wno-array-bounds -std=c++14 -MD -MT CMakeFiles/sipp.dir/src/call_generation_task.cpp.o -MF CMakeFiles/sipp.dir/src/call_generation_task.cpp.o.d -o CMakeFiles/sipp.dir/src/call_generation_task.cpp.o -c /buil│
d/sip-tester-3.7.3/src/call_generation_task.cpp                                                                                                                                                                   │
In file included from /usr/include/wolfssl/openssl/aes.h:32,                                                                                                                                                      │
                 from /build/sip-tester-3.7.3/include/jlsrtp.hpp:28,                                                                                                                                              │
                 from /build/sip-tester-3.7.3/include/rtpstream.hpp:22,                                                                                                                                           │
                 from /build/sip-tester-3.7.3/include/actions.hpp:33,                                                                                                                                             │
                 from /build/sip-tester-3.7.3/include/scenario.hpp:31,                                                                                                                                            │
                 from /build/sip-tester-3.7.3/include/sipp.hpp:67,                                                                                                                                                │
                 from /build/sip-tester-3.7.3/src/call_generation_task.cpp:33:                                                                                                                                    │
/usr/include/wolfssl/wolfcrypt/settings.h:2955:14: error: #warning "For timing resistance / side-channel attack prevention consider using harden options" [-Werror=cpp]                                           │
 2955 |             #warning "For timing resistance / side-channel attack prevention consider using harden options"                                                                                               │
      |              ^~~~~~~                                                                                                                                                                                      │
cc1plus: all warnings being treated as errors    
```

**Comment by orgads** [comment]
Fixed in 40610258b97c1d3da38713073b56c112944be9ac. Can you use it as a patch until next release?

**Comment by linuxmaniac** [comment]
Sure, thanks!
